js/src/wp-seo-post-scraper.js

```diff
@@ -1,9 +1,13 @@
-/* global YoastSEO, tinyMCE, wp, ajaxurl, wpseoPostScraperL10n, YoastShortcodePlugin, YoastReplaceVarPlugin, console */
+/* global YoastSEO, tinyMCE, ajaxurl, wpseoPostScraperL10n, YoastShortcodePlugin, YoastReplaceVarPlugin, console, require */
 (function( $ ) {
 	'use strict';
 
 	var currentKeyword = '';
 
+	// Initialize the keyword tab module.
+	var keywordTab;
```

Comment:
My issue is the readability of this. I am not debating the convention, I'm debating the readability of the code. I have just discussed this with @atimmer and we concluded that it might be best to rename `keywordTab` to `mainKeywordTab`.